src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs

```diff
@@ -345,5 +345,9 @@ public List<UcasNoteText> ReadNoteText(string folder)
 
             return result;
         }
+
+        private string CampusCodeFixer(string code){
+            return code.Replace(" ", "-").Trim();
```

Comment:
return code.Trim().Replace(" ", "-");

Reply:
return code.Trim().Replace("", "-");

Reply:
one or both of those

Reply:
optimise later

Reply:
lets hope `code` != buggy 

we should be a little bit more careful and null check